web_src/css/themes/theme-gitea-auto.css

```diff
@@ -1,2 +1,6 @@
 @import "./theme-gitea-light.css" (prefers-color-scheme: light);
 @import "./theme-gitea-dark.css" (prefers-color-scheme: dark);
+
+gitea-theme-meta-info {
+  --theme-display-name: "Auto";
+}
```

Comment:
Put it on `:root` please.

Reply:
If it is put in `:root`, the parser would be quite complex.

I do not see any problem by putting our privately defined variables into a privately declared element, it is valid CSS.

Reply:
Parses can simply be regex `--theme-display-name: (.+)`. I strongly prefer this to be in root, along with `--is-dark-theme` which is another already-existing metadata variable. Also I recommend name `--display-name`, the `theme` prefix is unneeded.

Reply:
No it doesn't work as that easy looking.

You could try to implement your idea to see whether it could be better and easier to maintain.

Reply:
In any case, I can not accept such a selector for an element that does not exist.